### ISSUE TITLE
Crash 32-bit builds

### ISSUE THREAD

**Issue by katyo** [DelSkayn/rquickjs#15]
Tests crashes with segmentation fault.

Linux i686:
https://github.com/katyo/rquickjs/runs/1406430612?check_suite_focus=true#step:8:29

Windows i686:
https://github.com/katyo/rquickjs/runs/1406430844?check_suite_focus=true#step:8:44

**Comment by katyo** [comment]
Seems generated bindings is platform dependent.
@DelSkayn How you invoke bindgen to generate bindings?
Should we add corresponding rules to our workflow?

**Comment by DelSkayn** [comment]
We could probably do something with the build script, I just wanted to avoid requiring users to have bindgen installed. 
As I had not yet done much investigation into automating it I just ran bindgen from the commandline and output the source to the file in rquickjs-sys. 

Regarding 32bit.
A lot of functions for dealing with quickjs's JSValues are defined in the header and inlined, as such bindgen generates no bindings for them.
So I implemented them myself in rust in the sys crate.
However, these functions differ between 32bit and 64bit as the 32bit platform uses NaN-tagging instead of a tagged union.
I only implemented the 64bit versions of the functions which is probably why any 32-bit platform version of the library will segfault
